artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/util/SyncCalculation.java

```diff
@@ -193,18 +291,29 @@ private static SequentialFileFactory newFactory(File datafolder, boolean datasyn
 
          case NIO:
             factory = new NIOSequentialFileFactory(datafolder, 1).setDatasync(datasync);
-            ((NIOSequentialFileFactory) factory).disableBufferReuse();
+            if (bufferReuse) {
+               ((NIOSequentialFileFactory) factory).enableBufferReuse();
```

Comment:
This is also wrong... you are changing the semantic of this test...


We are measuring how much write the system can do... Using the buffer will defeat the purpose of the test...


Besides you are using the timeouts from where? this is not the intention of this calculation.